internal/auth/repository_ent.go

```diff
@@ -32,25 +32,25 @@ func (e entRepository) FindUserByEmail(ctx context.Context, email string) (*enti
 	return u, nil
 }
 
-func (e entRepository) FindJWTRevocation(ctx context.Context, jti string) (*entity.JWTRevocation, error) {
-	r, err := e.client.JWTRevocation.Query().Where(jwtrevocation.Jti(jti)).Only(ctx)
+func (e entRepository) FindInstitutionInviteLinkWithInstitution(ctx context.Context, code string) (*entity.InstitutionInviteLink, error) {
```

Comment:
Hm, alternatively a case could be made that for functions like this, maybe eager loading should be avoided altogether. Two calls to the DB doesn't seem that bad

Reply:
Decided to access `Edge` directly instead